main.py

```
@@ -103,90 +115,108 @@ async def _handle_network_data(data):
                 )
 
                 block.nonce = block_data.get("nonce", 0)
                 block.hash = block_data.get("hash")
 
-                if chain.add_block(block):
-                    logger.info("Received block added to chain: #%s", block.index)
+                chain.add_block(block)
 
-        except Exception:
-            logger.exception("Error processing network data: %s", data)
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)